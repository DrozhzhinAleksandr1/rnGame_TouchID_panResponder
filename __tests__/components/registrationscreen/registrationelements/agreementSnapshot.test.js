import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { AgreementBtn } from '../../../../app/components/registrationscreen/registrationelements/agreementbtn';

it('create snapshot AgreementBtn', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<AgreementBtn />);
    expect(toolbarSnapshot).toMatchSnapshot();
});