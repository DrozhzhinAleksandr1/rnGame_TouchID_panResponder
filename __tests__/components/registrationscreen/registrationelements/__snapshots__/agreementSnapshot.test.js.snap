// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot AgreementBtn 1`] = `
<Component
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "marginBottom": 25,
      "width": "100%",
    }
  }
>
  <Component
    style={
      Object {
        "flexDirection": "row",
        "height": 20,
        "width": "100%",
      }
    }
  >
    <CheckBox />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "marginLeft": 5,
          },
          Object {
            "color": "#ffffff",
            "fontFamily": "Roboto-Light",
            "fontSize": 14,
            "lineHeight": 16,
            "textAlignVertical": "bottom",
          },
        ]
      }
    >
       
    </Text>
  </Component>
  <TouchableWithoutFeedback
    accessibilityLabel="BtnAgreementRegistrationID"
    onPress={[Function]}
    testID="BtnAgreementRegistrationID"
  >
    <Component>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "textAlign": "left",
              "textDecorationLine": "underline",
            },
            Object {
              "color": "#ffffff",
              "fontFamily": "Roboto-Light",
              "fontSize": 14,
              "lineHeight": 16,
              "textAlignVertical": "bottom",
            },
          ]
        }
      >
         
      </Text>
    </Component>
  </TouchableWithoutFeedback>
</Component>
`;
